repose-aggregator/tests/performance-tests/Jenkinsfile

```diff
@@ -27,14 +27,17 @@ stage("Performance Test") {
     def performanceJenkinsJob = "performance-test"
     def performanceTestParam = "perf_test"
     def extraOptionsParam = "extra_options"
+    def random = new Random()
 
     def jobsToRun = [:]
 
     for (int index = 0; index < perfTests.size(); index++) {
         def perfTest = perfTests[index]
         jobsToRun[perfTest] = {
             retry(3) {
-                build(job: performanceJenkinsJob, parameters: [
+                build(job: performanceJenkinsJob,
+                      quietPeriod: (random.nextInt() % 585) + 15,
```

Comment:
Side note, I'm glad there is a parameter for the `quietPeriod`. I think we tried to configure the job itself, but I did not realize that was something we could set here. That works quite well.